plugins/commando.c

```diff
@@ -640,6 +640,8 @@ static struct command_result *handle_custommsg(struct command *cmd,
 	size_t len;
```

Comment:
This commit message (GH reviews can't comment on commit messages unf!) needs to stand alone, e.g.

commando: save runes to datastore as we generate them, and start a listrunes command.

It could also be split into two commitments: the first saves them, the second lists them.  It's not vital, but it makes review simpler.